packages/core/src/scp-cf/client-credentials-token-cache.ts

```diff
@@ -21,7 +22,11 @@ const ClientCredentialsTokenCache = (
     credentials: ClientCredentials,
     token: ClientCredentialsResponse
   ): void => {
-    cache.set(getGrantTokenCacheKey(url, credentials), token, token.expires_in);
+    cache.set(
+      getGrantTokenCacheKey(url, credentials),
+      token,
+      token.expires_in && moment().add(token.expires_in, 'second').unix() * 1000
```

Comment:
> The number of seconds until the access token expires.

see doc: https://docs.cloudfoundry.org/api/uaa/version/74.27.0/index.html#authorization-code-grant-2

Reply:
What does the `token.expires_in && moment().add(token.expires_in, 'second').unix() * 1000` do? you want to return the value now+expires time if the `token.expires_in` is present right? I see that the test is working but I find this hard to read because `&&` is returning a boolean?

Reply:
`token.expires_in && moment().add(token.expires_in, 'second').unix() * 1000` equals `token.expires_in ?moment().add(token.expires_in, 'second').unix() * 1000 : token.expires_in`
Since the `expires_in` is typed `expires_in: number;`, I'll make it simple.